pages/osx/log.md

```diff
@@ -7,14 +7,14 @@
 
 `log stream`
 
-- Stream logs sent to syslog for PID 235:
+- Stream logs sent to syslog from the process with a specific PID:
 
 `log stream --process {{235}}`
 
-- Show logs sent to syslog for process named 'dasd':
+- Show logs sent to syslog from a process with a specific name:
 
 `log show --predicate "process == '{{dasd}}'"`
 
-- Dump all logs for the past hour:
+- Export all logs to disk for the past hour:
```

Comment:
From the docs:
> If an output path is not specified, system_logs.logarchive will be created in the current directory.

But I think we should add the `--output` flag to avoid confusion